### PR TITLE
Add email authenticator tutorial

### PR DESCRIPTION
**Pull request by erjohnson**
Tutorial on adding an (out-of-band) email authenticator

### CODE REVIEW

File: articles/multifactor-authentication/api/email.md [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+
+# Associate an Email Authenticator
+
+In this tutorial, you'll learn how to configure your application so users can use email authenticators.
+
+<%= include('./_authenticator-before-start') %>
+
+## 1. Enable email authenticators with the Management API
+
+Start by enabling email authenticators with the [Management API](/api/management). To do this, make a `PUT` request to the [/api/v2/guardian/factors/email](/api/management/v2/#!/Guardian/put_factors_by_name) endpoint. You'll need a [Management API Token](/api/management/v2/tokens) to perform the request:
```
Comment (cabralmartin) [inline articles/multifactor-authentication/api/email.md]: Can we add the required scopes for the access token?

File: articles/multifactor-authentication/api/email.md [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+```
+
+If successful, you'll receive a response like this:
+
+```json
+{
+  "authenticator_type": "oob",
+  "oob_channel": "email",
+  "recovery_codes": [ "N3BGPZZWJ85JLCNPZBDW6QXC" ],
+  "oob_code": "ata6daXAiOi..."
```
Comment (cabralmartin) [inline articles/multifactor-authentication/api/email.md]: Is `oob_code` part of the actual response? What is its purpose?

File: articles/multifactor-authentication/api/email.md [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+}
+```
+
```
Comment (cabralmartin) [inline articles/multifactor-authentication/api/email.md]: Can we add a p describing what happens next? ie. The users receives an email on their inbox ... grabs the code and puts it back into the app ... enrollment is completed.

Comment (cabralmartin) [reply articles/multifactor-authentication/api/email.md]: We should also add a note describing how the email template can be customized.

File: articles/multifactor-authentication/api/email.md [hunk 4]
```diff
@@ -0,0 +1,147 @@
...
+contentType:
+  - how-to
+useCase:
+  - customize-mfa
+---
+
+# Associate an Email Authenticator
+
+In this tutorial, you'll learn how to configure your application so users can use email authenticators.
+
```
Comment (cabralmartin) [inline articles/multifactor-authentication/api/email.md:18]: Need to add an info or warning box here describing that email MFA only works with this API and is not supported as part of the hosted mfa page. This means that users enrolled for email can only use it as an authenticator with apps that consume the API and not with those that redirect the user to the hosted page.

**Review by cabralmartin — CHANGES_REQUESTED** [review]

**Review by khorne3 — DISMISSED** [review]

File: articles/multifactor-authentication/api/email.md [hunk 5]
```diff
@@ -0,0 +1,154 @@
...
+```json
+{
+  "authenticator_type": "oob",
+  "oob_channel": "email",
+  "recovery_codes": [ "N3BGPZZWJ85JLCNPZBDW6QXC" ]
+}
+```
+
+Next the user should receive an email containing the code. Once they enter the code in your application, enrollment of the email authenticator will be complete.
```
Comment (cabralmartin) [inline articles/multifactor-authentication/api/email.md]: > Once they enter the code in your application, enrollment of the email authenticator will be complete.

What does this mean? What is the application supposed to do with the code they get from the user?

**Review by cabralmartin — CHANGES_REQUESTED** [review]

**Review by cabralmartin — APPROVED** [review]

**Pull request merged**